profile-service/src/main/resources/db/migration/stage/V9.21__revert_user_designated_body_for_stage.sql

```diff
@@ -0,0 +1,13 @@
+UPDATE `UserDesignatedBody` set `designatedBodyCode` = '1-1P9Y9QH' where `designatedBodyCode` = '1-1RSG4X0';
```

Comment:
As on Reference db, the old codes "1-AIIDQQ" & "1-AIIDNQ" won't exist. We should revert to the test codes.